RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -23,14 +27,29 @@
 particleBasedIsolationTmp.valueMapElePFblockIso = cms.string("gedGsfElectronsTmp")
 
 egmPhotonIsolationCITK = _egmPhotonIsolationAOD.clone()
+egmElectronIsolationCITK = _egmElectronIsolationCITK.clone()
+egmElectronIsolationPileUpCITK = _egmElectronIsolationCITKPileUp.clone()
 
-for iPSet in IsoConeDefinitionsTmp:
+#change particleBasedIsolation object to tmp
+for iPSet in IsoConeDefinitionsPhotonsTmp:
```

Comment:
ok, but there is this line:

from RecoEgamma.EgammaIsolationAlgos.egmIsoConeDefinitions_cfi import IsoConeDefinitions  as IsoConeDefinitionsPhotonsTmp

does this mean that when you do:

from Bla import A as B

and in this case A and B reference the same object?